.pre-commit-config.yaml

```diff
@@ -0,0 +1,63 @@
+default_language_version:
+  python: python3
+exclude: ^.*\b(migrations)\b.*$
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.1.0
```

Comment:
There's a new version to use: 22.3.0
This will avoid an import error from click on some python versions:
```
ImportError: cannot import name '_unicodefun' from 'click'
```

Reply:
Yes updated!